Backend/starter_project_zsilentcoders/BlogApp3/BlogApp.API/Controllers/ReviewController.cs

```diff
@@ -0,0 +1,66 @@
+﻿using BlogApp.Application.Features._Indices.CQRS.Commands;
+using BlogApp.Application.Features._Indices.CQRS.Queries;
+using BlogApp.Application.Features._Indices.DTOs;
+using BlogApp.Application.Features.Review.CQRS.Command;
+using BlogApp.Application.Features.Review.CQRS.Commands;
+using BlogApp.Application.Features.Review.CQRS.Queries;
+using BlogApp.Application.Features.Review.DTOs;
+using BlogApp.Application.Responses;
+using BlogApp.Domain;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+
+// For more information on enabling Web API for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace BlogApp.API.Controllers
+{
+    [Route("api/[Controller]")]
+    [ApiController]
+    public class ReviewController : ControllerBase
```

Comment:
Extend from baseController and adjest your response accordingly(You can reference how the blog controller is implemented)